### PR TITLE
Foxy release versions 2022-09-28

### PR DESCRIPTION
**Pull request by jacobperron**
Announcement: https://discourse.ros.org/t/preparing-for-foxy-sync-and-patch-release-2022-09-27/27464
Changes: https://github.com/orgs/ros2/projects/36

CI:
* Linux [Build Status](https://ci.ros2.org/job/ci_linux/17399/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/11898/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/17867/)
* CentOS [Build Status](https://ci.ros2.org/view/All/job/ci_linux-rhel/255/)
  * Baseline: https://ci.ros2.org/view/All/job/ci_linux-rhel/183/

Packaging:
* Linux [Build Status](https://ci.ros2.org/job/ci_packaging_linux/523/)
* Linux-aarch64 [Build Status](https://ci.ros2.org/job/ci_packaging_linux-aarch64/98/)
* Windows[Build Status](https://ci.ros2.org/job/ci_packaging_windows/205/)
* Windows Debug [Build Status](https://ci.ros2.org/job/ci_packaging_windows/206/)
* CentOS [Build Status](https://ci.ros2.org/job/ci_packaging_linux-rhel/34/)

### CODE REVIEW

File: ros2.repos [hunk 1]
```diff
@@ -2,23 +2,23 @@ repositories:
...
+    version: 0.9.7
   ament/ament_package:
     type: git
     url: https://github.com/ament/ament_package.git
     version: 0.9.5
   ament/google_benchmark_vendor:
     type: git
     url: https://github.com/ament/google_benchmark_vendor.git
-    version: 0.0.7
+    version: 0.0.3
```
Comment (nuclearsandwich) [inline ros2.repos:21]: Is this number supposed to have gone down instead of up?

File: ros2.repos [hunk 2]
```diff
@@ -63,10 +63,6 @@ repositories:
     type: git
     url: https://github.com/ros-tooling/libstatistics_collector.git
     version: 1.0.1
-  ros-tracing/ros2_tracing:
-    type: git
-    url: https://gitlab.com/ros-tracing/ros2_tracing.git
-    version: 1.0.5
```
Comment (nuclearsandwich) [inline ros2.repos:69]: Is ros2_tracing intentionally removed?

Comment (nuclearsandwich) [reply ros2.repos:69]: Ah, I see it moved to https://github.com/ros2/ros2/pull/1327/files#diff-0b86fcc230a228fb210653f2069d07ee0ab117da02c6471640ae12327835ff4fR298-R301

**Review by nuclearsandwich — APPROVED** [review]
One number goes down instead of up (comment inline) but other than that looks okay.

File: ros2.repos (hunk 1 above)
Comment (jacobperron) [reply ros2.repos:21]: Strange.. Looking at the `foxy` branch, 0.0.3 seems correct: https://github.com/ament/google_benchmark_vendor/commits/foxy

Comment (jacobperron) [reply ros2.repos:21]: doh, I guess I accidentally bumped this during the last couple patch releases, but regular Foxy CI has been using the `foxy` branch,

Comment (clalancette) [reply ros2.repos:21]: I just looked, and it is 0.0.3 in https://github.com/ros/rosdistro/blob/e76b1707a4f1eea0e782c53ea2bdd87769d837a7/foxy/distribution.yaml#L1609-L1620 as well.

Looking through history, it looks like this was bumped to 0.0.7 in #1240, but that was bumping it up from 0.0.6.  It looks like the issue here came in originally during #1176, which was about a year ago.

Regardless, I think this is meant to be 0.0.3, and it was just somehow marked wrong in the last 2 patch releases.

**Review by clalancette — APPROVED** [review]

File: ros2.repos (hunk 1 above)
Comment (jacobperron) [reply ros2.repos:21]: 0.0.3 is consistent with rosdistro: https://github.com/ros/rosdistro/blob/e76b1707a4f1eea0e782c53ea2bdd87769d837a7/foxy/distribution.yaml#L1609-L1614

**Pull request merged**